quickwit/quickwit-proto/protos/quickwit/ingester.proto

```diff
@@ -268,6 +271,20 @@ message DecommissionRequest {
 message DecommissionResponse {
 }
 
+message MrecordlogSummaryRequest {
```

Comment:
```suggestion
message MRecordLogSummaryRequest {
```

if clippy does not whine too much...

Reply:
the messages can be renamed (2 upper cases is fine iirc). The rpc could be, but this leads to (imo ugly) `async fn m_recordlog_summary(..)`, which we could do, but could also not.